### ISSUE TITLE
[DataCap Application] <Limradar><open data>

### ISSUE THREAD

**Issue by Pauline546** [fidlabs/Open-Data-Pathway#103]
### Data Owner Name

Shanghai Limradar Electronic Technology Co., Ltd

### Data Owner Country/Region

China

### Data Owner Industry

Information, Media & Telecommunications

### Website

https://www.limradar.com/

### Social Media Handle

Limingruida

### Social Media Type

WeChat

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

500TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f1dqbbvnmv2mxd4itmuiked5sureutb46fsyh743q

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
Shanghai Limradar was established in 2020. Our company focuses on radar detection, multi-sensor fusion, artificial intelligence traffic control and other technology research in the field of transportation, and provides relevant millimeter wave radar products around intelligent transportation, intelligent security, advanced assisted driving, rail safety and other applications.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
The data we store are radar test records used in various fields. Our company's product---Radar can be applied in several scenarios, including Smart Traffic, Smart Security, Advanced Driving Assistance, Track and traffic safety, etc. We have a large accumulation of data.
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you are a data preparer. What is your location (Country/Region)

China

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
We use lotus to prepare data.
```

### Has this dataset been stored on the Filecoin network before? If so, please explain and make the case why you would like to store this dataset again to the network. Provide details on preparation and/or SP distribution.

```text
We used to store some data in filecoin. This time we put new data into filecoin.
```

### Please share a sample of the data

```text
https://1drv.ms/u/s!AqB4q1Q7IpiSa6AKXvWDW-wEGeE?e=sVB5mO
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Weekly

### For how long do you plan to keep this dataset stored on Filecoin

1.5 to 2 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, South America, Europe

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Big Data Exchange

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f03192503, HongKong
f03233966, Zhejiang
f03156722, USA
f03216485, Sichuan
f01975299, Guangdong
```

### How do you plan to make deals to your storage providers

Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by martapiekarska** [comment]
Hi @Pauline546! 
According to the policy on our pathway:

1. The data stored must be available via spark checking, and through manual retrieval attempts by the pathway operators
2. The data must be accessible (for retrieval) from at least two distinct geographic locations
3. The data must be retrievable for free for anyone that wants to access it.
4. There must be clear documentation provided on how the data is prepared / transformed between the source data set and the data stored in Filecoin
5. Deals / Pieces stored to Filecoin should be full - there should not be significant 'slack' or 'padding' in stored data
6. Individual semantic items of the data should be able to be individually referenced / accessed.
7. When a piece is downloaded from the stored dataset, there must be a way to identify what part of the original dataset that piece represents and confirm that it is indeed valid data from the dataset. (this could be e.g. provided via a log file indicating the mapping of offsets / files into stored deals)

Can you please document how will you ensure these are met please?

**Comment by Pauline546** [comment]
@martapiekarska 
Yes, we will check the retrieval results of sps via spark dashboard. And we'll be finding sps in different regions to work with together.
<img width="819" alt="1d44" src="https://github.com/user-attachments/assets/fdebd3b2-c0c3-484b-973d-0a0503c51057">


We use our own code to package and deal with the files. This ensures that the layer of file catalogs is preserved while packing the dataset into car files. This is convenient for us to find the files in the dataset according to the file catalogs after retrieval, which can be done to verify the correctness of the data. Besides that, we will record the PieceCID in our database in relation to the original file.

**Comment by martplo** [comment]
@Pauline546
The links you provided contain around 200MB of data. What exactly does the full dataset contain? Since you apply for open data pathway, we'd like to browse all the data that you're planning to store.

**Comment by willscott** [comment]
> We use our own code to package and deal with the files. This ensures that the layer of file catalogs is preserved while packing the dataset into car files. This is convenient for us to find the files in the dataset according to the file catalogs after retrieval, which can be done to verify the correctness of the data. Besides that, we will record the PieceCID in our database in relation to the original file.

How does anyone else find the files in the dataset / get access to the file catalogs?
this is an *open* data pathway, so it can't be a tool only you have access to

**Comment by dampud** [comment]
@Pauline546 Please let us know if you intend to continue the application.

**Comment by dampud** [comment]
The thread is being closed due to inactivity.
